Vagrantfile

```diff
@@ -15,9 +15,27 @@ Vagrant.configure("2") do |config|
   config.vm.provision "shell", inline: <<-SHELL
     cat << EOF | dnf -y shell
 update
-install podman make golang-go libseccomp-devel bats jq
+install podman make golang-go libseccomp-devel bats jq \
+ patch protobuf protobuf-c protobuf-c-devel protobuf-compiler \
+ protobuf-devel protobuf-python libnl3-devel libcap-devel libnet-devel \
+ nftables-devel libbsd-devel gnutls-devel
```

Comment:
Should the `# TODO: remove this after criu 3.14 is released` be noted for the additionall installs here as well?

Reply:
Yes, but this is very minor and it can be easily seen from the commit itself.

Besides, it is complicated to insert a comment in the middle of a single shell command.